Dockerfile

```diff
@@ -0,0 +1,40 @@
+# Chalice dockerfile
```

Comment:
This was moved from backend/Dockerfile to the top level directory because we needed access to the tests directory. In an ideal world for Docker purposes, the tests directory would be moved to be inside of backend, but we won't tackle that in this PR.